Description: remove icon link from example html file
 This patch removes a suposedly privacy breach detected by lintian.
 As the Trolltech site is down and the example is simply not working I am
 just removing the link to it.
Author: Lisandro Damián Nicanor Pérez Meyer <lisandro@debian.org>
Forwarded: not-needed
Last-Update: 2014-06-25

---
 examples/xml/htmlinfo/trolltech_com.html |    3 ---
 1 file changed, 3 deletions(-)

--- a/examples/xml/htmlinfo/trolltech_com.html
+++ b/examples/xml/htmlinfo/trolltech_com.html
@@ -110,9 +110,6 @@
         <style type="text/css" media="all">@import url(http://trolltech.com/IEFixes.css);</style>
     <![endif]-->
 
-    <link rel="shortcut icon" type="image/x-icon"
-          href="http://trolltech.com/favicon.ico" />
-
     <link rel="home" href="http://trolltech.com"
           title="Front page" />
     <link rel="search"
